src/test/kotlin/sirius/biz/storage/s3/ObjectStoresTest.kt

```diff
@@ -0,0 +1,117 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.storage.s3
+
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import sirius.biz.storage.layer2.BlobStorage
+import sirius.db.jdbc.OMA
+import sirius.kernel.BaseSpecification
+import sirius.kernel.SiriusExtension
+import sirius.kernel.commons.Files
+import sirius.kernel.commons.Streams
+import sirius.kernel.commons.Tuple
+import sirius.kernel.di.std.Part
+import java.io.File
+import java.io.FileOutputStream
+import java.net.URL
+
+import java.nio.charset.StandardCharsets
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+@ExtendWith(SiriusExtension::class)
+class ObjectStoresTest {
+
+    @Test
+    fun `createBucketWorks`() {
+        lateinit var download: File
```

Comment:
We we need this as lateinit?

Reply:
we indeed do not need it, I changed the variable initialisation. Thank you!